### ISSUE TITLE
[Documentation]: Should we add Returns section to the docstrings of our get methods (get_metadata, etc.)

### ISSUE THREAD

**Issue by pauladkisson** [catalystneuro/neuroconv#1097]
See #1063

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by pauladkisson** [comment]
On the one hand, documentation = good, OTH simple 1-liners usually suffice to convey all the relevant info.

**Comment by h-mayorquin** [comment]
I am leaning toward yes as in most cases implementing them at the base interface should work for most cases so it is not a lot of work. Plus, is a uniform rule, everything should have arguments and returns...

**Comment by h-mayorquin** [comment]
@pauladkisson is your preference for yes? I think that the other team members do not have a preference so if you have this one we can just start moving slowly towards it as I don't have a strong one either.

**Comment by pauladkisson** [comment]
Yeah, let's do it. It will be more consistent with all the other docstrings.

**Comment by pauladkisson** [comment]
> Maybe seeing if it can add this is a good test bed for Cline capabilities?

Yeah, good idea! It's a simple request/easy to validate, but requires keeping track of a lot of files, which has been a failure point for agents in my experience. Do we have a CN API key? It might be a larger job.
